bigtable-hbase-1.x-parent/bigtable-hbase-1.x/src/main/java/com/google/cloud/bigtable/hbase1_x/BigtableAdmin.java

```diff
@@ -131,32 +126,31 @@ public boolean isSnapshotFinished(HBaseProtos.SnapshotDescription snapshot)
 
   @Override
   public void restoreSnapshot(String s, boolean b, boolean b1)
-      throws IOException, RestoreSnapshotException {}
+      throws IOException, RestoreSnapshotException {
+    throw new UnsupportedOperationException("restoreSnapshot"); // TODO
+  }
 
   @Override
   public void cloneSnapshot(String s, TableName tableName, boolean b)
-      throws IOException, TableExistsException, RestoreSnapshotException {}
+      throws IOException, TableExistsException, RestoreSnapshotException {
+    if (!b) {
+      this.cloneSnapshot(s, tableName);
+    }
+    throw new UnsupportedOperationException("cloneSnapshot"); // TODO
+  }
 
   /** {@inheritDoc} */
   @Override
   public List<HBaseProtos.SnapshotDescription> listSnapshots() throws IOException {
-    ListSnapshotsRequest request =
-        ListSnapshotsRequest.newBuilder().setParent(getSnapshotClusterName().toString()).build();
-
-    ListSnapshotsResponse snapshotList =
-        Futures.getChecked(tableAdminClientWrapper.listSnapshotsAsync(request), IOException.class);
+    List<String> backups =
+        Futures.getChecked(
```

Comment:
Same as above, getChecked prevents interruption...I dont think this is something that we want